SPECS/k3s/k3s.spec

```diff
@@ -103,6 +103,9 @@ exit 0
 * Tue Aug 30 2022 Animesh Garg <animeshgarg@microsoft.com> - 1.24.3-1
 - Updated k3s to 1.24.3
 
+* Tue Aug 30 2022 Anuj Garg <anujgarg@microsoft.com> - 1.25.5-2
```

Comment:
The most recent changelog item should go at the top

Reply:
Sure. Some issue happened with this PR so closing this for now and will create a new one